### PR TITLE
Update OpenSSL to 1.1.1g

### PR DESCRIPTION
**Pull request by KVS85**
- Linux: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-matrix/925/
- Windows: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-windows-matrix/1086/
- macOS: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-mac-matrix2/2/

oskar PR: https://github.com/arangodb/oskar/pull/242

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,8 @@
 v3.6.4 (XXXX-XX-XX)
 -------------------
 
+* Updated OpenSSL to 1.1.1f.
```
Comment (joerg84) [inline CHANGELOG:4]: There is already the same changelog entry for v3.6.3

Comment (joerg84) [reply CHANGELOG]: Fixed with s/f/g

**Comment by KVS85** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-ANY-mac-matrix2/5/

**Comment by KVS85** [comment]
Tests blue except unrelated failures.

**Pull request merged**